reference_material/data_dictionary

```diff
@@ -0,0 +1,130 @@
+# Current available fields:
+
+## api_endpoint_id:
+The unique identifier for the API endpoint.
+Type: numeric
+## api_id:
+The ID of the API.
+Type: numeric
+
+## api_vendor_id:
+The ID of the API vendor.
+Type: numeric
+
+## api_vendor
+The vendor of the API, e.g "OneLook", "Victoria State Government Australia", "TeejLab"...
+Type: string
+
+## api:
+The name of the API, e.g "Datamuse Dictionary", "DepartmentAPI"...
+Type: string
+
+## category
+The category of the API, e.g "Research & Education", "AI & Data Science"...
+Type: string
+
+## usage_base:
```

Comment:
Perhaps we can add:
- Free: there are free limits, such as the number of calls that can be made for free. These limits are also available to hackers
- - Commercial: need registration. Additional layer of security